src/TuDa.CIMS.Web/Components/AssetList.razor.cs

```diff
@@ -0,0 +1,123 @@
+using Microsoft.AspNetCore.Components;
+using TuDa.CIMS.Shared.Entities;
+
+
+namespace TuDa.CIMS.Web.Components;
+
+public partial class AssetListPartialClass
```

Comment:
🔧 Rename that back to `AssetList`

Reply:
If I rename it back to `AssetList` I get the errors back. Then the methods and the parameter are not recognised.